src/test/ui/cmse-nonsecure-entry/params-on-stack.rs

```diff
@@ -0,0 +1,11 @@
+// gate-test-cmse_nonsecure_entry
+// compile-flags: --target thumbv8m.main-none-eabi --crate-type lib
+// only-thumbv8m.main-none-eabi
+#![feature(cmse_nonsecure_entry)]
+#![no_std]
+
+#[no_mangle]
+#[cmse_nonsecure_entry]
+pub extern "C" fn entry_function(a: u32, b: u32, c: u32, d: u32, e: u32) -> u32 {
```

Comment:
We should also have a test that does the same thing, except with just 4 arguments (as a build-pass test).

Reply:
Added such test!